src/annotator/integrations/test/html-metadata-test.js

```diff
@@ -405,5 +405,24 @@ describe('HTMLMetadata', () => {
 
       assert.equal(doc.uri(), canonicalLink.href);
     });
+
+
+    it('should log an error if URI is not decodable', () => {
+      // Arrange
+      // '%%CUST_ID%%' is an invalid escape sequence, so it will throw a URIError when decoded
+      const badURI = 'https://example.com/?foo=%%CUST_ID%%' 
+      sinon.stub(testDocument, '_getDocumentHref').returns(badURI); 
+      const consoleErrorSpy = sinon.spy(console, 'error');
+
+      // Act
+      testDocument.uri(); // The method that triggers the try...catch
```

Comment:
The comment is redundant.

```suggestion
      testDocument.uri();
```

Reply:
Also, it would be interesting to assert on what this method returns when parsing the URI fails. Right now this test is only asserting on console.error being invoked.